CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-### 0.7.1 (Next)
+### 0.8.0 (Next)
 
 * [#208](https://github.com/mongoid/mongoid-history/pull/208): Fix: history tracks fields declared after `track_history` - [@mikwat](https://github.com/mikwat).
+* [#211](https://github.com/mongoid/mongoid-history/pull/211): BREAKING CHANGE - Enable tracking create/destroy by default - [@jagdeepsingh](https://github.com/jagdeepsingh).
```

Comment:
Lets not be dramatic :) It's not a breaking change, it's just a different behavior. I would just remove the BREAKING CHANGE part and call it a day.